### PR TITLE
install leaflet and display map

### PR DESCRIPTION
**Pull request by seb879**
install leaflet and react-leaflet dependencies and display the map

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css"
```
Comment (erabin05) [inline public/index.html]: The indentation could be better

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+    zoom: 14,
+  };
+
+  render() {
+    return (
+      <div>
+        <Map center={this.state.center} zoom={this.state.zoom}>
+          <TileLayer
+            attribution='&amp;copy <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+            url="https://maps.heigit.org/openmapsurfer/tiles/roads/webmercator/{z}/{x}/{y}.png"
```
Comment (erabin05) [inline src/App.js]: single (attribution) or double (url)  quotes, make your choice. What AirBnb says ?

**Review by erabin05 — CHANGES_REQUESTED** [review]
Just syntax and indentation rectification.
Doesn't your linter say something ?

File: src/App.js (hunk 2 above)
Comment (seb879) [reply src/App.js]: attribution deleted and corrected

File: public/index.html (hunk 1 above)
Comment (seb879) [reply public/index.html]: I have corrected the indentation

**Review by seb879 — COMMENTED** [review]
Pull request corrected

**Review by erabin05 — APPROVED** [review]
OK

**Pull request merged**